Library/Homebrew/brew.sh

```diff
@@ -12,10 +12,28 @@ then
   then
     export LC_ALL="en_US.UTF-8"
   else
-    export LC_ALL="C.UTF-8"
+    utf8_locales=$(locale -a | grep -e \.utf8 -e \.UTF-8)
+    if [[ -n $utf8_locales && 
+         ($utf8_locales =~ (^|[^a-zA-Z0-9_])(C|en_US)\.(utf8|UTF-8) ||
+          $utf8_locales =~ ^[a-zA-Z0-9_.]*) ]]
+    then
+      export LC_ALL=$BASH_REMATCH
+    elif [[ -n $(locale -a | grep -w C) ]]
```

Comment:
```suggestion
    elif [[ -n $(locale -a | grep --word-regexp C) ]]
```

and as many other long flags as possible would make this a bit easier to follow.